setup.py

```diff
@@ -47,6 +47,23 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
+try:
+    import django
+    os.environ.setdefault(
+        "DJANGO_SETTINGS_MODULE",
+        "analytical.tests.settings"
+    )
+    django.setup()
+except ImportError:
+    print(
+        "Could not import django."
+        "This is fine, unless you intend to run unit tests"
+    )
+except AttributeError:
+    # django.setup() does not exist on older djangos, so don't worry about
+    # that.
+    pass
```

Comment:
Good. Though, actually, I think it should better fail (e.g. explicitly via `UnsupportedVersionError('Django < 1.7 not supported')`) instead of silencing it alltogether. As a goody the comment would then be unneeded.

A comment in Python usually reveals that there's something not as explicit or obvious as it should be.